package.json

```diff
@@ -85,6 +85,7 @@
     "url": "git://github.com/kuzzleio/kuzzle.git"
   },
   "devDependencies": {
+    "trace": "^3.1.1",
```

Comment:
should be in the dependencies, since it's used in kuzzle's code and not exclusively in tests